fronts-client/src/lib/dnd/Level.tsx

```diff
@@ -72,9 +66,8 @@ interface OuterProps<T> {
   onDrop: (e: React.DragEvent, to: PosSpec) => void;
   renderDrag?: (data: T) => React.ReactNode;
   renderDrop?: (props: DropProps) => React.ReactNode | null;
-  // Any occurence of these in the data transfer will cause all dragging
-  // behaviour to be bypassed.
-  denylistedDataTransferTypes?: string[];
+  // If this function returns false, all dragging behaviour is bypassed.
+  denyDragEvent?: (e: React.DragEvent) => boolean;
```

Comment:
This is now a function because we no longer what consumers will have to match on until render time.